tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/workspace/NewLiferayWorkspaceWizard.sdef

```diff
@@ -68,6 +68,25 @@
             <margin-vertical>0</margin-vertical>
             <margin>0</margin>
         </page>
+        <page>
+            <id>LiferayWorkspaceSettingWizardPage</id>
+            <label>Liferay Workspace Setting</label>
+            <description>Liferay Workspace Setting</description>
+            <content>
+                <property-editor>Environment</property-editor>
```

Comment:
WE need to somehow give more info than this.  Just having the properties there it is hard for new user to figure out what these are doing.